platforms/documentation/docs/src/samples/ide/problems-api-usage/groovy/sample-ide/build.gradle

```diff
@@ -15,7 +15,7 @@ application {
 
 dependencies {
     implementation('reporters:model-builder-plugin')
-    implementation('org.gradle:gradle-tooling-api:8.6-milestone-1')
+    implementation('org.gradle:gradle-tooling-api:8.9')
```

Comment:
❔  Is there a way for us to make those updated on every release?

Reply:
Initially I wanted to do the same thing, but it's not simple. I'll follow-up on this in a separate PR.